sw/device/lib/testing/test_framework/ottf_isrs.c

```diff
@@ -192,6 +193,12 @@ void ottf_timer_isr(uint32_t *exc_info) {
 OT_WEAK
 bool ottf_console_flow_control_isr(uint32_t *exc_info) { return false; }
 
+OT_WEAK
+bool ottf_alert_isr(uint32_t *exc_info) {
+  ottf_generic_fault_print(exc_info, "Alert IRQ", ibex_mcause_read());
```

Comment:
Maybe print some info about the alert that was triggered before the `abort` to make debugging easier?